### ISSUE TITLE
Questions on evaluation

### ISSUE THREAD

**Issue by zzaebok** [apple/ml-kg-mt#1]
Thanks for your great work!

I have a few questions on evaluation.

1. Did you use `sacrebleu` to get BLEU score?
2. If you used `sacrebleu`, which tokenizer is used to get the result from Thai language? I found that using default `13a` tokenizer didn't reproduce the result on the paper (Much lower score). Instead, I could see the similar result when I use `char` tokenizer.
3. Score on the paper is calculated on `test` split in references?
4. Can you share the prompt used in GPT series?

**Comment by c-simone** [comment]
Hi, thanks for your interest in our work!

1. Yes, we used `sacrebleu`.
2. We used `char`.
3. The scores on the paper are computed on `all`. However, for future work, we are looking into dividing `all` into three splits: `sample` (e.g., for fewshot), `validation` (e.g., for model selection), and `test` (for the testing the final models). The new `test` split should be large enough, and the new `sample` and `validation` splits should allow easier development in settings that are different from the one we used in the paper.
4. Replace the variables within curly brackets:
    * The system prompt for GPT4 was similar to the following: `You are an expert translator and your task is to translate from {source_locale} to {target_locale}.`
    * The queries were formatted as following: `{source_locale}: "{source_text}"\n{target_locale}: "`
    
Hope this answers your questions!
